### PR TITLE
Test against multiple  Python and Django versions with TravisCI

### PR DESCRIPTION
**Pull request by Duologic**
I heard we were using this in a Python 3/Django 1.11 project, but it was not tested against these versions. Here we go then.

### CODE REVIEW

**Comment by Duologic** [comment]
@RoelandMatthijssens Can you have a look?

Just let me know when I can merge, I'll also push the wheel to PyPi in that case.

File: requirements/requirements_test.txt [hunk 1]
```diff
@@ -32,32 +42,25 @@ flake8-mock==0.3
...
-
-# Coverage uploads
-coveralls
-coverage==3.7.1
-docopt==0.6.2
-requests==2.9.1
+flake8-user-model==1.1
 
 # StatsD allows us to push monitoring data to a Graphite server
-git+ssh://git@github.com/vikingco/django-statsd.git@0.4.0#egg=django-statsd-mozilla
```
Comment (RoelandMatthijssens) [inline requirements/requirements_test.txt]: Don't we want statsd in the actual requirements instead of test_requirements?

Comment (Duologic) [reply requirements/requirements_test.txt]: Looks like an optional requirement to me: https://github.com/vikingco/django-triggers/search?utf8=%E2%9C%93&q=statsd&type=

Comment (RoelandMatthijssens) [reply requirements/requirements_test.txt]: :+1:

**Comment by RoelandMatthijssens** [comment]
Looks good. :ship:

**Pull request merged**